integration-tests/metrics-v4.2.19.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -e -u -o pipefail
+
+test_name="$(basename "${0}" .sh)"
+project=metrics
+repository=https://github.com/dropwizard/metrics.git
+revision=v4.2.19
+
+if [ "${#}" -gt 2 ] || ([ "${#}" = 2 ] && [ "${1:---sync}" != '--sync' ]); then
+  echo "Usage: ${0} [--sync] [<report_directory>]"
+  exit 1
+fi
+do_sync="$([ "${#}" = 0 ] || [ "${1:-}" != '--sync' ] || echo 1)"
+report_directory="$([ "${#}" = 0 ] || ([ -z "${do_sync}" ] && echo "${1}") || ([ "${#}" = 1 ] || echo "${2}"))"
+
+# XXX: We exclude the CollectorMutability and the Immutable*|Preconditions*|StringRules.StringIsNullOrEmpty refaster rules
```

Comment:
```suggestion
# XXX: We exclude the `CollectorMutability` and the `Immutable*|Preconditions*|StringRules.StringIsNullOrEmpty` refaster rules
```

Reply:
Instead, we also have this flag: https://github.com/PicnicSupermarket/error-prone-support/blob/master/error-prone-contrib/src/main/java/tech/picnic/errorprone/bugpatterns/util/ThirdPartyLibrary.java#L43 which is used in the CollectorMutability: https://github.com/PicnicSupermarket/error-prone-support/blob/master/error-prone-contrib/src/main/java/tech/picnic/errorprone/bugpatterns/CollectorMutability.java#L61 😉.